pillar/base/haproxy.sls

```diff
@@ -94,6 +94,11 @@ haproxy:
       verify_host: salt.psf.io
       check: "GET /.well-known/acme-challenge/sentinel HTTP/1.1\\r\\nHost:\\ salt.psf.io"
 
+    publish-files:
+      domains: []
```

Comment:
this should be set to a list containing the domain we are serving from (`salt-public.psf.io`), otherwise requests won't be routed to it. the empty domains block above is for lets-encrypt which needs to respond to any domain.